src/test/scala/chiselTests/Reg.scala

```diff
@@ -51,6 +51,16 @@ class ShiftTester(n: Int) extends BasicTester {
   }
 }
 
+class ShiftMemTester(n: Int, dp_mem: Boolean) extends BasicTester {
+  val (cntVal, done) = Counter(true.B, n)
+  val start = 23.U
+  val sr = ShiftRegister(cntVal + start, n, true.B, dp_mem, Some("simple_sr"))
```

Comment:
looking a this test, should we call this flavor of `ShiftRegister` `ShiftMem`?

Reply:
Believe that this is in connection to the subsequent comment.